lib/collections/accounts.js

```diff
@@ -0,0 +1,113 @@
+import { SimpleSchema } from "meteor/aldeed:simple-schema";
+import { shopIdAutoValue } from "./helpers";
+import { Address } from "./address";
+import { Metafield } from "./metafield";
+
+/**
+ * Accounts Schemas
+ */
+
+export const Profile = new SimpleSchema({
```

Comment:
If we are just modifying the `Account` schema, we should import these from `lib/schemas` rather than copying them over.